spec/sidekiq/terms_of_use/sign_up_service_updater_job_spec.rb

```diff
@@ -162,9 +174,11 @@
           end
           let(:mpi_profile) { build(:mpi_profile, icn: mpi_icn, sec_id:, given_names:, family_name:) }
           let(:mpi_icn) { 'some-mpi-icn' }
+          let(:status) { { opt_out: false, agreement_signed: false } }
 
           before do
             allow(Rails.logger).to receive(:info)
+            allow(service_instance).to receive(:status).and_return(status)
```

Comment:
I think here the `allow(service_instance)...` mock will happen in the `before do` above since this is within that context above

Reply:
Do you mean to move it up into the context above?